src/api/app/helpers/webui/request_helper.rb

```diff
@@ -115,54 +115,72 @@ def diff_label(diff)
     "#{diff['project']} / #{diff['package']} (rev #{diff['rev']})"
   end
 
-  # rubocop:disable Style/FormatString
+  # rubocop:disable Style/FormatString, Metrics/PerceivedComplexity
```

Comment:
`Metrics/PerceivedComplexity` is addressed once the `FIXME` I added is dealt with.